pkg/document/values.go

```diff
@@ -180,6 +182,10 @@ func createValueRowsFromList(
 	// documented without descriptions
 	if hasDescription {
 		jsonableObject := convertHelmValuesToJsonable(values)
```

Comment:
Reverse these so we only produce the jsonableObject in the case that we are documenting objects

Reply:
I've reworked a little so that the documentation output now shows the correct "type" value - previously my change meant it said "string" for lists / objects